netapp_dataops_traditional/netapp_dataops/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM alpine:latest
```

Comment:
This docker and wrapper script can be used to run the traditional module without the need to install all dependencies on the server. 
This can be useful for environments that will not allow internet connectivity to pip repositories. 

Anyway I removed it in the last commit.